app/views/dashboards/show.html.erb

```diff
@@ -24,6 +24,18 @@
             <span class="label label-info" id="pull-requests-count">
               <%= pull_requests.count %>
             </span>
+            <% if pull_requests.count == 1 %>
+              <% tweetstring = t("dashboard.tweet_progress_singular") %>
+            <% else %>
+              <% tweetstring = t("dashboard.tweet_progress_plural", count: pull_requests.count) %>
+            <% end %>
+            <% user_url = "https://24pullrequests.com/users/#{user.nickname}" %>
```

Comment:
No hard-coding of hosts. I think you're looking for `user_url` that already should exist.